<?xml version="1.0"?>
<launch>
    <include file="$(find assignment2_1_rt)/launch/sim_w1.launch" />
    <param name="target_x" value= "0.0" />
    <param name="target_y" value= "1.0" />
    <node pkg="assignment2_1_rt" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment2_1_rt" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment2_1_rt" type="bug_as.py" name="bug_action_service" output="screen" />

    <!-- <node pkg="assignment2_1_rt" type="action_client_node_A.py" name="bug_action_client" output="screen" /> -->
    <node pkg="assignment2_1_rt" type="coordinates_srv.py" name="coordinates_service" output="screen" />
    
</launch>
